queries/objdump/highlights.scm

```diff
@@ -0,0 +1,20 @@
+(address) @number
+(byte) @constant
+(hexadecimal) @number
+(integer) @number
+(section_address) @number
+
+(disassembly_section_label) @text.title
+[(file_offset) (discriminator)] @field
+(file_path) @string
+(identifier) @namespace
+(instruction) @text
+(label) @label
+(machine_code_bytes) @field
```

Comment:
```suggestion
```

Reply:
In retrospect this is obvious, I don't know why I didn't see it initially. Thank you for pointing it out!